Public/Src/Cache/ContentStore/Library/Stores/IContentDirectory.cs

```diff
@@ -10,6 +10,7 @@
 using BuildXL.Cache.ContentStore.Interfaces.Time;
 using BuildXL.Cache.ContentStore.Interfaces.Tracing;
 using BuildXL.Cache.ContentStore.UtilitiesCore;
+using static BuildXL.Cache.ContentStore.Stores.FileSystemContentStoreInternal;

```

Comment:
Why?

Reply:
This was from a previous iteration, forgot to remove.

Reply:
By the way, in addition to threading comments, when you resolve a comment, feel free to mark resolved.

---
In reply to: [285284696](https://github.com/microsoft/BuildXL/pull/329#discussion_r285284696) [](ancestors = 285284696)